pkg/util/trivy/trivy.go

```diff
@@ -251,24 +253,23 @@ func (c *Collector) ScanDockerImage(ctx context.Context, imgMeta *workloadmeta.C
 	}
 
 	if c.config.overlayFSSupport && fanalImage.inspect.GraphDriver.Name == "overlay2" {
-		return c.scanOverlayFS(ctx, fanalImage, imgMeta, scanOptions)
+		var layers []string
+		if layerDirs, ok := fanalImage.inspect.GraphDriver.Data["LowerDir"]; ok {
```

Comment:
With Docker, `GraphData` contains info about overlayfs. It's not the case with containerd.